Week 3 MVP and Week 2 in-progress code + Previous Pull Request

GROD8962 opened the pull request:
Ignore previous pull request, use this one instead.

GROD8962 committed:
Update README.md

New attempt.

GROD8962 committed:
Update pom.xml

GROD8962 committed:
Update script.js

GROD8962 committed:
Update index.html

GROD8962 committed:
Create blog.html

GROD8962 committed:
Create images.html

GROD8962 committed:
Create .gitignore

GROD8962 committed:
Add files via upload

GROD8962 committed:
Delete .gitignore

GROD8962 committed:
Update HelloWorldServlet.java

Tutorial step.

GROD8962 committed:
Update index.html

Tutorial step.

GROD8962 committed:
Update index.html

Week 2 javascript changes

GROD8962 committed:
Update script.js

Week 2 javascript changes

GROD8962 committed:
Update HelloWorldServlet.java

Week 2 javascript changes

GROD8962 committed:
Update HelloWorldServlet.java

sonambenakatti reviewed (approved):
Not necessary, it would be neat to include in the description any photos/videos of the changes made!

sonambenakatti commented:
Think the review will go through once I have write access :)

GROD8962 committed:
Week 3 MVP, Week 2 in-progress code

Implements a simple hardcoded chart (Week 3 MVP), includes incomplete code for Week 2's project.

GROD8962 committed:
Week 3 MVP

Implements a simple hardcoded chart for LEGO BIONICLE set distribution per year.

GROD8962 committed:
Week 2 in-progress code

Creates a basic template for the POST request, still a WIP.

GROD8962 commented:
Added more to this pull request since for whatever reason GitHub didn't merge it before. I looked around, and as far as I understand, the invitation should give you write access? I'm not sure why the opposite is happening if it's indicating you that you don't have write access. Please excuse my ignorance if I'm missing something obvious.
